ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/picklist/activity-picklist-remote-auto-complete-options.component.ts

```diff
@@ -28,6 +29,7 @@ import { BaseActivityPicklistQuestion } from './baseActivityPicklistQuestion.com
             [placeholder]="block.picklistLabel"
             [matAutocomplete]="autoCompleteFromSource"
             (keyup)="onInput($event.target.value)"
+            [ngModel]="getInitialAnswerOptionLabel()"
```

Comment:
Let's try to avoid using ngModel. The rest of the code is in the style that of using component events and rxjs to handle input and output. I don't know what this is supposed to do. Please figure out how we can initialize by setting the value for component directly.

Reply:
As an option you can init the BehaviourSubject (`searchValue$`) in `ngOnInit` method.
Like:
```
  ngOnInit(): void {
        const initSearchValue = ...;
        this.searchValue$ = new BehaviorSubject(initSearchValue);
        ...
}
```

Reply:
Thank you